### ISSUE TITLE
Xtend Quickfix should also remove the unused imports and all writing accesses to the field

### ISSUE THREAD

**Issue by miklossy** [eclipse/xtext-xtend#736]
Given is the following Xtend file:

```
package foo

import java.util.Date
import java.util.List

class A {
  List<Date> a
  
  new() {
    a = newArrayList
  }
}
```
Currently, the Xtend Quickfix Remove unused member only removes the declaration of an unused field
![screencast](https://user-images.githubusercontent.com/20393472/53564454-42c65b00-3b57-11e9-97ba-d9afda5450d3.gif)

The Quickfix should clean up as much as possible, such as:

- the import statements that become unused after removing the unused field
- all writing accesses to the field

@loradd  Would you like to take care of this issue? Would be awesome...

**Comment by miklossy** [comment]
**Hint:** The JDT removes 'a', but keeps the assignments with side-effects.

**Comment by loradd** [comment]
Ok, I'll take care of it!

**Comment by cdietrich** [comment]
@miklossy is this fixed now?

**Comment by miklossy** [comment]
Thanks for your contribution! I merged yor PR into the master branch! This issue is fixed in 2.18.
